contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -9,11 +9,21 @@ import {UpgradeableClaimable} from "../common/UpgradeableClaimable.sol";
 contract TrueCreditAgency is UpgradeableClaimable {
     using SafeERC20 for ERC20;
 
+    mapping(ITrueFiPool2 => bool) public isPoolAllowed;
+
+    event PoolAllowanceChanged(ITrueFiPool2 pool, bool isAllowed);
```

Comment:
I don't know if I like "Allowance", it kind of resembles ERC20 allowance. Might be a little misleading or maybe it is just my perception